src/mento.ts

```diff
@@ -1,26 +1,26 @@
-import { BigNumber, Signer, providers } from 'ethers'
 import {
   IBroker,
   IBroker__factory,
   IExchangeProvider,
   IExchangeProvider__factory,
 } from '@mento-protocol/mento-core-ts'
+import { BigNumber, providers, Signer } from 'ethers'
+import { Address } from './types'
 import {
   getBrokerAddressFromRegistry,
   getSymbolFromTokenAddress,
   increaseAllowance,
 } from './utils'
-import { Address } from './types'
 
 import { strict as assert } from 'assert'
 
-interface Exchange {
+export interface Exchange {
   providerAddr: Address
-  id: Address
-  assets: string[]
+  id: string
```

Comment:
I guess you are changing this because even though it's an address it's more like an identifier rather than a meaningful address?

Reply:
Its not an address though right? It's a hash which is longer than an address

Reply:
ah yes you are right, I confused this id with something else